content/developer/reference/backend/orm.rst

```diff
@@ -785,7 +785,13 @@ Fields/Views
 
 .. automethod:: Model.fields_get
 
-.. automethod:: Model.fields_view_get
+.. automethod:: Model.get_view
+
+.. versionadded:: saas-15.4
+
+    :meth:`~odoo.models.Model.fields_view_get()` is now deprecated.
+    It returns the same content, but is not used anymore by the web client.
+    Please use :meth:`~odoo.models.Model.get_view()` instead.
```

Comment:
@AntoineVDV warning or versionadded ? :D

Reply:
Can't we do something [like this](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated)? We don't currently use `versionadded` because we would do it so inconsistently it's better not to mention it at all.

Reply:
It's a bit the same display as versionadded, but the text is more adequate :+1:

Reply:
Sure but it's relative to the deprecated method, which makes it easier to maintain IMO.